backend/src/src-licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/outputGenerators/DocxGenerator.java

```diff
@@ -194,9 +197,9 @@ private void fillOwnerGroup(XWPFDocument document, Project project) throws XmlEx
     }
 
     private void fillAttendeesTable(XWPFDocument document, Project project) throws XmlException, TException {
-        XWPFTable table = document.getTables().get(0);
+        XWPFTable table = document.getTables().get(OVERVIEW_TABLE_INDEX);
```

Comment:
seems also to me that `OVERVIEW_TABLE_INDEX` is at least not declared in the same class, so javac telling me it is missing this sounds likely.